Detectors/MUON/MID/Raw/include/MIDRaw/ELinkDataShaper.h

```diff
@@ -0,0 +1,70 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MIDRaw/ELinkDataShaper.h
+/// \brief  MID e-link data shaper
```

Comment:
would not hurt to add a brief description of what the shaping actually is / is doing ?

Reply:
Done